database/test-schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "users": {
+    "user_id_1": {
+      "brooklyn_completed_tasks": "",
+      "email": "taylor.swift@scl.com",
+      "manhattan_completed_tasks": {
+        "id_13": true
+      },
+      "name": "Bob",
```

Comment:
We don't currently have a name in the signup flow, I'm not sure we need this

Reply:
hmm ok sure. its like saved and put back from the Auth User object but if theres a pattern of "hello X" then this can be saved in the data model. otherwise its a +0 from me whether its added or not since its going to be avail in the data model no matter what from the auth callback

Reply:
Let's leave, and if it's empty it doesn't really matter